lms/static/sass/elements/_controls.scss

```diff
@@ -404,7 +404,7 @@
     }
   }
 
-  .btn-link {
+  .btn-secondary {
```

Comment:
@marcotuts I will add btn-link class to Leave Team button. Should i use the same `btn-secondary` for edit team button or do we have any existing rule for that?